src/System.Diagnostics.Process/tests/ProcessCollectionTests.cs

```diff
@@ -59,6 +59,7 @@ public void TestModuleCollectionBehavior()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "https://github.com/dotnet/corefx/issues/22174")]
```

Comment:
I think the comment will be better and we can mention issue # in the comment too